src/main/java/net/datafaker/providers/healthcare/CareProvider.java

```diff
@@ -0,0 +1,30 @@
+package net.datafaker.providers.healthcare;
+
+import net.datafaker.providers.base.AbstractProvider;
+
+/**
+ * Generate random types of health care providers
+ *
+ * @since 2.3.0
+ */
+public class CareProvider extends AbstractProvider<HealthcareProviders> {
+    protected CareProvider(HealthcareProviders faker) {
+        super(faker);
+    }
+
+    /**
+     * Generate random hospital name
+     * @return A hospital name
```

Comment:
Why do we need javadoc which just repeats the actual method name here and in the rest?
IMHO: if javadoc can not bring more info than existing name then probably we don't need that javadoc

Reply:
Agreed

Reply:
Yeah, I agree too. However existing codebase is inconsistent, cause in some places there are such dummy javadocs, in other not. So I preferred to add it cause I didn't know which one you prefer.

So I will remove those.